dev/ci/docker_linux/Dockerfile

```diff
@@ -110,21 +110,20 @@ RUN /root/patch_firebase.sh
 
 # Install dashing
 # This is why we need golang installed.
```

Comment:
nit: This comment is wrong now.

Reply:
In what way?

Reply:
Oh sorry, I didn't see it in context...fixing